ts/parts/SVGLabel.ts

```diff
@@ -0,0 +1,548 @@
+/* *
+ *
+ *  (c) 2010-2020 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import type SVGRenderer from './SVGRenderer';
+import SVGElement from './SVGElement.js';
+import U from './Utilities.js';
+const {
+    defined,
+    extend,
+    isNumber,
+    merge,
+    removeEvent
+} = U;
+
+/**
+ * SVG label to render text.
+ * @private
+ * @class
+ * @name Highcharts.SVGLabel
+ * @augments Highcharts.SVGElement
+ */
+class SVGLabel extends SVGElement {
+
+    /* *
+     *
+     *  Static Properties
+     *
+     * */
+
+    public static readonly emptyBBox: Highcharts.BBoxObject = { width: 0, height: 0, x: 0, y: 0 };
+
+    /* *
+     *
+     *  Constructors
+     *
+     * */
+
+    public constructor(
+        renderer: SVGRenderer,
+        str: string,
+        x: number,
+        y?: number,
+        shape?: Highcharts.SymbolKeyValue | string,
+        anchorX?: number,
+        anchorY?: number,
+        useHTML?: boolean,
+        baseline?: boolean,
+        className?: string
+    ) {
```

Comment:
Did you consider decreasing number of params? Either, `label(renderer, string, position, options)` or simple `label(renderer, options)` ?

Perhaps `anchorX/Y` could be removed from init (unless these options are crucial when initializing) and moved to `setAnchor()` method?

Reply:
Yes I considered that, but forgot. I think I'll do that. Makes it more future proof.

Reply:
I think this will make sense:
`label(string, x, y, options)` 

Then it can be aligned directly with `text(string, x, y, options)`. 

Will create a separate PR for that, so we have better comparison.